templates/static_analysis/android_binary_analysis.html

```diff
@@ -126,6 +126,10 @@ <h3 class="box-title">App Score</h3>
           </br>
           <span class="label label-danger">VirusTotal Detection</span> <b>{{ VT_RESULT.positives }}/{{ VT_RESULT.total }}</b>
         {% endif %}
+        {% if trackers %}
+          </br>
+          <span class="label label-danger">Trackers Detection</span> <b>{{ trackers.detected_trackers }}/{{ trackers.total_trackers }}</b>
```

Comment:
how about `label-danger` ?

Reply:
@ajinabraham which one sounds the best for You ?

Reply:
sorry` label-warning`

Reply:
done :) and yep better to put only a warning